osi_featuredata.proto

```diff
@@ -54,17 +54,18 @@ message DetectionHeader
     //
     // The origin of the detection coordinate system (= spherical polar
     // coordinates) is the sensor mounting pose in the host vehicle coordinate
-    // system.
+    // system w.r.t. orientation of the sensor mounting position.
     //
     // The host vehicle coordinate system is at the center of the rear axle
     // (from the top view as well as from the side view) in the static case.
     // The origin represents the current mounting pose to the best knowledge of
-    // the sensor. It includes the estimated 6D pose estimation given by  the
+    // the sensor. It includes the estimated 6D pose estimation given by the
     // calibration. The uncertainty of this estimation is given with the
     // corresponding 6D root mean squared error. The estimation of the current
     // origin does not include effects due to short-time dynamics, such as pitch
     // angles from braking, but includes long-time calibration values, such as
     // pitch angles from luggage in the trunk.
+    // 
     optional MountingPosition mounting_position = 4;
```

Comment:
Even after multiple reads I do not understand the definition in lines 47 through 67.

It seems that the "mounting position" not only covers the position, but also the orientation. Consider renaming... "MountingPosition6DoF" for instance... Ist that what the term "pose" is supposed to mean here? The combination of position and orientation?

Furthermore, the definition of the reference coordinate system (the host vehicle coordinate system) is ambiguous. It could be defined as follows:
``The host vehicle coordinate system is right-handed and body-fixed. Its xz-plane is the vehicle's plane of symmetry. In static equilibrium on horizontal ground, its y-axis is collinear with the rear axle, pointing left, and the z-axis is pointing vertically upwards.``
With this definition, "mounting position" (i.e., position and orientation) can be said to be defined "with respect to the host vehicle cooridnate system" without any ambiguity.

Reply:
I made a reference to OSI::Vehicle. So we need not to redefine the host vehicle coordinate system. 

    // Mounting position of the sensor (origin and orientation of the cartesian
    // sensor coordinate system); given relative from the middle of the rear
    // axle of the host vehicle coordinate system (see: \c OSI::Vehicle vehicle
    // reference point) to the sensor detection coordinate system's origin. The
    // sensor detection coordinate system is a spherical polar coordinate system
    // (same origin and orientation as cartesian sensor coordinate system).
    //
    // The sensor detection coordinate system's x-axis must be the angle
    // bisector of the sensor's horizontal and vertical field of view in a
    // right-handed coordinate system. The xy-plane represents the horizontal
    // field of view and the xz-plane the vertical field of view.
    //
    // The origin represents the current mounting pose to the best knowledge of
    // the sensor. The estimation of the 6D pose given by the calibration. The
    // uncertainty of this estimation is given with the corresponding 6D root
    // mean squared error. The estimation of the current origin does not include
    // effects due to short-time dynamics, such as pitch angles from braking,
    // but includes long-time calibration values, such as pitch angles from
    // luggage in the trunk.